<prc-local-header menu="results"></prc-local-header>

<div id="content" role="main" class="grid_24 clearfix view-animate-container">
    <!-- If you require a breadcrumb trail, its root should be your service.
    You don't need a breadcrumb trail on the homepage of your service... -->
    <nav id="breadcrumb">
        <p>
            <a href="#/">PRIDE Cluster</a> &gt;
            Result Files
        </p>
    </nav>

    <section class="alpha grid_17">
        <h2 id="spInt" class="icon icon-functional" data-icon="=">Cluster Results Files</h2>

        <p class="help-content">
            These cluster results files are derived from the PRIDE Cluster results. They classified the PRIDE Cluster results
            in different files that can be use by the community for different analyses and aims. For example the Unidentified clusters
            with at least 100 spectra can be use to look for new peptide idientifications, variants, modifications that are difficult
            to identified with common search engine strategies; the Phospho clusters contains all reliable phosphorilated clusters.
        </p>


    </section>

    <!--<section class="omega grid_7">-->
        <!--<h3 class="icon icon-functional" data-icon="M">Topics</h3>-->
        <!--<ul id="secondary-nav" class="help-content">-->
            <!--&lt;!&ndash;TODO: Add active menu class="active" and ideally move the menu when you click&ndash;&gt;-->
            <!--<li>-->
                <!--<a ng-click="jumpTo('latRel')">Latest Release</a>-->
            <!--</li>-->
            <!--<li class="last">-->
                <!--<a ng-click="jumpTo('spLib')">How to use the libraries</a>-->
            <!--</li>-->
        <!--</ul>-->
    <!--</section>-->


    <!--<h3 id="latRel">Latest Release (Release: {{ latestRelease.releaseDate }}, Version: {{ latestRelease.version }})</h3>-->
    <!--<section class="grid_6 omega">-->
        <!--&lt;!&ndash;Aspera application object&ndash;&gt;-->
        <!--<div class="aspera-content">-->
            <!--<object id="aspera-web" type="application/x-aspera-web" width="0" height="0"></object>-->
        <!--</div>-->

        <!--<h5 ng-show="asperaplugincheck()">-->
            <!--<a href="http://www.asperasoft.com/connect/" class="icon icon-functional" data-icon="=">-->
                <!--Fast Download with (<a href="http://asperasoft.com/">Aspera</a>)-->
            <!--</a>-->
        <!--</h5>-->
    <!--</section>-->

    <section class="grid_24">


        <section class="animate-repeat" ng-repeat="spectrumLibrary in latestRelease.spectrumLibraries">

            <div class="grid_8 spectral-library">
                <div class="grid_24">
                    <!--<div class="grid_4 alpha">-->
                        <!--<img class="species-img" alt="Test" ng-src="{{ './img/' + spectrumLibrary.taxonomyId + '.svg' }}"-->
                             <!--title="Download">-->
                    <!--</div>-->
                    <div class="grid_24">
                        <h4>{{ resultFile.name }}</h4>
                    </div>

                </div>
                <div class="grid_24">
                    <ul>
                        <li>
                            Common name: {{ resultFile.description}}
                        </li>
                        <li>
                            File Size: {{ bytesToSize(spectrumLibrary.fileSize) }}
                        </li>
                    </ul>
                </div>
                <div class="grid_20 push_2 links">
                    <div class="grid_12" ng-repeat="downloadUrl in spectrumLibrary.downloadURLs">

                        <a ng-if="downloadUrl.protocol == 'FTP'"
                           class="button"
                           ng-href="{{downloadUrl.url}}">
                            {{downloadUrl.protocol}} Download
                        </a>
                        <a ng-if="downloadUrl.protocol == 'ASPERA'"
                           ng-hide="asperaplugincheck()"
                           class="button"
                           style="cursor: pointer"
                           ng-href=""
                           ng-click="asperaDownload(downloadUrl.url)">
                            {{downloadUrl.protocol}} Download
                        </a>
                    </div>

                </div>
            </div>
        </section>
    </section>


    <!--<section class="grid_18 alpha">-->
        <!--<h3 id="spLib">How to use the libraries?</h3>-->

        <!--<p class="help-content">PRIDE spectrum libraries do not contain protein annotations. Instead, you should use-->
            <!--your spectrum search-->
            <!--engine's inbuilt mapping function to map the identified peptides to a protein database of your choice.-->
        <!--</p>-->

        <!--<h4>Spectra ST</h4>-->

        <!--<p class="help-content">1. Convert the library into SpectraST's binary format</p>-->

<!--<pre>-->
<!--<code class="help-content">-->
    <!--# replace filenames with current version of PRIDE Cluster libraries-->
    <!--spectrast -cNpride_human-2012_06 pride_human-2012_06.msp-->
<!--</code>-->
<!--</pre>-->


        <!--<p class="help-content">2. Search the library and add protein annotations (threshold of 3.0 m/z, search all-->
            <!--charge states,-->
            <!--load library into memory)-->
        <!--</p>-->

<!--<pre>-->
<!--<code class="help-content">-->
    <!--spectrast -sR -sM3.0 -sz -sLpride_human-2012_06.splib -sDswissprot_human.fasta spectra.mgf-->
<!--</code>-->
<!--</pre>-->
        <!--<p class="help-content">For more information on how to use SpectraST please refer to the SpectraST-->
            <!--<a href="http://tools.proteomecenter.org/wiki/index.php?title=Software:SpectraST" target="_blank">documentation</a>.-->
        <!--</p>-->

        <!--<h4>Pepitome</h4>-->

        <!--<p class="help-content">-->
            <!--Pepitome can read PRIDE Cluster's spectral libraries directly without the need for any file conversion.-->
            <!--Additionally, the re-mapping of peptide identifications to an updated protein database is also done by-->
            <!--default.-->
            <!--Therefore, no adaptions are necessary to use PRIDE Cluster's spectral libraries with Pepitome.-->
        <!--</p>-->

        <!--<p class="help-content">-->
            <!--For more information please refer to the Pepitome-->
            <!--<a href="http://forge.fenchurch.mc.vanderbilt.edu/scm/viewvc.php/*checkout*/trunk/doc/index.html?root=pepitome"-->
               <!--target="_blank">documentation</a>.-->
        <!--</p>-->
    <!--</section>-->

</div>
